browser/ui/content_settings/brave_autoplay_blocked_image_model.cc

```diff
@@ -10,6 +10,7 @@
 #include "brave/browser/ui/content_settings/brave_autoplay_content_setting_bubble_model.h"
 #include "brave/grit/brave_generated_resources.h"
 #include "brave/components/vector_icons/vector_icons.h"
+#include "components/content_settings/browser/page_specific_content_settings.h"
```

Comment:
do we need a dep for `//components/content_settings_browser`?

Reply:
Yes, will add!

Reply:
Addressed by fc0b4ac50a2c4f569624473bb156476207070962